app/controllers/vocab_sheets_controller.rb

```diff
@@ -51,4 +58,17 @@ def set_vocab_sheet
     def set_title
       @title = @sheet.name
     end
+
+    def build_rendered_pdf(html:)
+      renderer = PdfRenderingService.new(from_html: html)
+      renderer.render
+      renderer.pdf
```

Comment:
🐘 why separate out `renderer.render` and `renderer.pdf`, when `renderer.render` returns the pdf? 
`renderer.render` _could_ be renamed `renderer.get_pdf`, and line 65 could be removed. 🔬 in the scheme of things though : )

Reply:
hmm I kind of agree, although .pdf here ends up being a new instance of RenderedPdf while the render method runs everything required to get it to that point so I kind of like the separation.